### ISSUE TITLE
Running DLRM model got  Runtime error: cublas_status_not_supported

### ISSUE THREAD

**Issue by BookChan** [NVIDIA-Merlin/HugeCTR#26]
Follow by [dlrm_fp32_64k.json](https://github.com/NVIDIA/HugeCTR/blob/master/samples/dlrm/dlrm_fp32_64k.json),
we test DLRM in our  data: label_dim=1, dense_dim=5 slot_num=75  . And got error in the first fully connected layer.

What's wrong with my data or model config? Or there are some bugs in hugectr?

log
```
[10d12h27m36s][HUGECTR][INFO]: end_lr is not specified using default: 0.000000
[6421.34, init_end, ]
[6421.35, run_start, ]
HugeCTR training start:
[6421.36, train_epoch_start, 0, ]
[HCDEBUG][ERROR] Runtime error: cublas_status_not_supported /tmp/HugeCTR/HugeCTR/src/layers/fully_connected_layer.cu:143 

[HCDEBUG][ERROR] Runtime error: operation not permitted when stream is capturing /tmp/HugeCTR/HugeCTR/src/session.cpp:451 

[HCDEBUG][ERROR] Runtime error: cublas_status_not_supported /tmp/HugeCTR/HugeCTR/src/layers/fully_connected_layer.cu:143 

Terminated with error
```

# model config
```
{
  "solver": {
    "lr_policy": "fixed",
    "display": 1,
    "max_iter": 2,
    "gpu": [
        0
    ],
    "batchsize": 32,
    "snapshot": 1,
    "snapshot_prefix": "./tmp/daw",
    "eval_interval": 1,
    "batchsize_eval":32,
    "eval_metrics": [
        "AUC:0.9",
        "AverageLoss"
    ],
    "eval_batches": 1,
    "input_key_type": "I64"
},
"optimizer": {
    "type": "Adam",
    "global_update": false,
    "adam_hparam": {
        "learning_rate": 0.0001,
        "beta1": 0.9,
        "beta2": 0.999,
        "epsilon": 1e-08
    }
},
"layers": [
    {
        "name": "data",
        "type": "Data",
        "source": "./tmp/file_list.txt",
        "eval_source": "./tmp/file_list_test.txt",
        "check": "Sum",
        "label": {
            "top": "label",
            "label_dim": 1
        },
        "dense": {
            "top": "dense",
            "dense_dim": 5
        },
        "sparse": [
            {
                "top": "data1",
                "type": "DistributedSlot",
                "max_feature_num_per_sample": 180,
                "slot_num": 75
            }
        ]
    },
    {
        "name": "sparse_embedding1",
        "type": "DistributedSlotSparseEmbeddingHash",
        "bottom": "data1",
        "top": "sparse_embedding1",
        "sparse_embedding_hparam": {
            "max_vocabulary_size_per_gpu": 24000000,
            "load_factor": 0.75,
            "embedding_vec_size": 16,
            "combiner": 1
        }
    },
  
      {
        "name": "fc1",
        "type": "InnerProduct",
        "bottom": "dense",
        "top": "fc1",
         "fc_param": {
          "num_output": 512
        }
      },
  
   
    {
        "name": "relu1",
        "type": "ReLU",
        "bottom": "fc1",
        "top": "relu1" 
      },
  
      {
        "name": "fc2",
        "type": "InnerProduct",
        "bottom": "relu1",
        "top": "fc2",
         "fc_param": {
          "num_output": 256
        }
      },
  
      {
        "name": "relu2",
        "type": "ReLU",
        "bottom": "fc2",
        "top": "relu2"     
      },
      
      {
        "name": "fc3",
        "type": "InnerProduct",
        "bottom": "relu2",
        "top": "fc3",
         "fc_param": {
          "num_output": 16
        }
      },
  
      {
        "name": "relu3",
        "type": "ReLU",
        "bottom": "fc3",
        "top": "relu3"     
      },
      
      {
        "name": "interaction1",
        "type": "Interaction",
        "bottom": ["relu3", "sparse_embedding1"],
        "top": "interaction1"
      },
  
      {
        "name": "fc4",
        "type": "InnerProduct",
        "bottom": "interaction1",
        "top": "fc4",
         "fc_param": {
          "num_output": 1024
        }
      },
  
      {
        "name": "relu4",
        "type": "ReLU",
        "bottom": "fc4",
        "top": "relu4" 
      },
        
  
      {
        "name": "fc5",
        "type": "InnerProduct",
        "bottom": "relu4",
        "top": "fc5",
         "fc_param": {
          "num_output": 1024
        }
      },
  
      {
        "name": "relu5",
        "type": "ReLU",
        "bottom": "fc5",
        "top": "relu5"     
      },
      
      {
        "name": "fc6",
        "type": "InnerProduct",
        "bottom": "relu5",
        "top": "fc6",
         "fc_param": {
          "num_output": 512
        }
      },
  
      {
        "name": "relu6",
        "type": "ReLU",
        "bottom": "fc6",
        "top": "relu6"     
      },
  
      {
        "name": "fc7",
        "type": "InnerProduct",
        "bottom": "relu6",
        "top": "fc7",
         "fc_param": {
          "num_output": 256
        }
      },
  
      {
        "name": "relu7",
        "type": "ReLU",
        "bottom": "fc7",
        "top": "relu7"     
      },
      
      {
        "name": "fc8",
        "type": "InnerProduct",
        "bottom": "relu7",
        "top": "fc8",
         "fc_param": {
          "num_output": 1
        }
      },
      
      {
        "name": "loss",
        "type": "BinaryCrossEntropyLoss",
        "bottom": ["fc8","label"],
        "top": "loss"
      } 
    ]
  }
```

**Comment by BookChan** [comment]
it seems add one  dropout layer works:
```
{
  "solver": {
    "lr_policy": "fixed",
    "display": 1,
    "max_iter": 2,
    "gpu": [
        0
    ],
    "batchsize": 32,
    "snapshot": 1,
    "snapshot_prefix": "./tmp/daw",
    "eval_interval": 1,
    "batchsize_eval":32,
    "eval_metrics": [
        "AUC:0.9",
        "AverageLoss"
    ],
    "eval_batches": 1,
    "input_key_type": "I64"
},
"optimizer": {
    "type": "Adam",
    "global_update": false,
    "adam_hparam": {
        "learning_rate": 0.0001,
        "beta1": 0.9,
        "beta2": 0.999,
        "epsilon": 1e-08
    }
},
"layers": [
    {
        "name": "data",
        "type": "Data",
        "source": "./tmp/file_list.txt",
        "eval_source": "./tmp/file_list_test.txt",
        "check": "Sum",
        "label": {
            "top": "label",
            "label_dim": 1
        },
        "dense": {
            "top": "dense",
            "dense_dim": 5
        },
        "sparse": [
            {
                "top": "data1",
                "type": "DistributedSlot",
                "max_feature_num_per_sample": 180,
                "slot_num": 75
            }
        ]
    },
    {
        "name": "sparse_embedding1",
        "type": "DistributedSlotSparseEmbeddingHash",
        "bottom": "data1",
        "top": "sparse_embedding1",
        "sparse_embedding_hparam": {
            "max_vocabulary_size_per_gpu": 24000000,
            "load_factor": 0.75,
            "embedding_vec_size": 16,
            "combiner": 1
        }
    },
  
      {
        "name": "fc1",
        "type": "InnerProduct",
        "bottom": "dense",
        "top": "fc1",
         "fc_param": {
          "num_output": 512
        }
      },
  
   
    {
        "name": "relu1",
        "type": "ReLU",
        "bottom": "fc1",
        "top": "relu1" 
      },
  
      {
        "name": "fc2",
        "type": "InnerProduct",
        "bottom": "relu1",
        "top": "fc2",
         "fc_param": {
          "num_output": 256
        }
      },
  
      {
        "name": "relu2",
        "type": "ReLU",
        "bottom": "fc2",
        "top": "relu2"     
      },
      
      {
        "name": "fc3",
        "type": "InnerProduct",
        "bottom": "relu2",
        "top": "fc3",
         "fc_param": {
          "num_output": 16
        }
      },
  
      {
        "name": "relu3",
        "type": "ReLU",
        "bottom": "fc3",
        "top": "relu3"     
      },
      
      {
        "name": "interaction1",
        "type": "Interaction",
        "bottom": ["relu3", "sparse_embedding1"],
        "top": "interaction1"
      },
  
      {
        "name": "fc4",
        "type": "InnerProduct",
        "bottom": "interaction1",
        "top": "fc4",
         "fc_param": {
          "num_output": 1024
        }
      },
  
      {
        "name": "relu4",
        "type": "ReLU",
        "bottom": "fc4",
        "top": "relu4" 
      },
        
  
      {
        "name": "fc5",
        "type": "InnerProduct",
        "bottom": "relu4",
        "top": "fc5",
         "fc_param": {
          "num_output": 1024
        }
      },
  
      {
        "name": "relu5",
        "type": "ReLU",
        "bottom": "fc5",
        "top": "relu5"     
      },
      
      {
        "name": "fc6",
        "type": "InnerProduct",
        "bottom": "relu5",
        "top": "fc6",
         "fc_param": {
          "num_output": 512
        }
      },
  
      {
        "name": "relu6",
        "type": "ReLU",
        "bottom": "fc6",
        "top": "relu6"     
      },
  
      {
        "name": "fc7",
        "type": "InnerProduct",
        "bottom": "relu6",
        "top": "fc7",
         "fc_param": {
          "num_output": 256
        }
      },
  
      {
        "name": "relu7",
        "type": "ReLU",
        "bottom": "fc7",
        "top": "relu7"     
      },
          
      {
        "name": "dropout7",
        "type": "Dropout",
        "rate": 0.5,
        "bottom": "relu7",
        "top": "dropout7" 
      },
      {
        "name": "fc8",
        "type": "InnerProduct",
        "bottom": "dropout7",
        "top": "fc8",
         "fc_param": {
          "num_output": 1
        }
      },
      
      {
        "name": "loss",
        "type": "BinaryCrossEntropyLoss",
        "bottom": ["fc8","label"],
        "top": "loss"
      } 
    ]
  }
```

**Comment by Jianbing-D** [comment]
@BookChan  I tried with your model config (without dropout), and used datasets which are generated with `data_generator`. But did not reproduce this issue. There might be something wrong with your datasets. Could you please elaborate how you prepare your datasets?
```bash
[0, init_start, ]
HugeCTR Version: 2.2
Config file: issue_26.json
[24d01h14m36s][HUGECTR][INFO]: algorithm_search is not specified using default: 1
[24d01h14m36s][HUGECTR][INFO]: Algorithm search: ON
[24d01h14m38s][HUGECTR][INFO]: Peer-to-peer access cannot be fully enabled.
Device 0: Tesla V100-SXM2-16GB
[24d01h14m38s][HUGECTR][INFO]: Initial seed is 1712382765
[24d01h14m38s][HUGECTR][INFO]: cache_eval_data is not specified using default: 0
[24d01h14m38s][HUGECTR][INFO]: max_nnz is not specified using default: 180
[24d01h14m38s][HUGECTR][INFO]: num_internal_buffers 1
[24d01h14m38s][HUGECTR][INFO]: num_internal_buffers 1
[24d01h14m38s][HUGECTR][INFO]: max_vocabulary_size_per_gpu_=24000000
[24d01h14m41s][HUGECTR][INFO]: gpu0 start to init embedding
[24d01h14m42s][HUGECTR][INFO]: gpu0 init embedding done
[24d01h14m42s][HUGECTR][INFO]: warmup_steps is not specified using default: 1
[24d01h14m42s][HUGECTR][INFO]: decay_start is not specified using default: 0
[24d01h14m42s][HUGECTR][INFO]: decay_steps is not specified using default: 1
[24d01h14m42s][HUGECTR][INFO]: decay_power is not specified using default: 2.000000
[24d01h14m42s][HUGECTR][INFO]: end_lr is not specified using default: 0.000000
[5898.5, init_end, ]
[5898.52, run_start, ]
HugeCTR training start:
[5898.55, train_epoch_start, 0, ]
[24d01h14m42s][HUGECTR][INFO]: Iter: 1 Time(1 iters): 0.004367s Loss: 0.688271 lr:0.000100
[24d01h14m42s][HUGECTR][INFO]: Rank0: Dump hash table from GPU0
[24d01h14m42s][HUGECTR][INFO]: Rank0: Write hash table <key,value> pairs to file
[24d01h14m42s][HUGECTR][INFO]: Done
[5915.71, eval_start, 0.5, ]
[24d01h14m42s][HUGECTR][INFO]: Evaluation, AUC: 0.582722
[5917.88, eval_accuracy, 0.582722, 0.5, 1, ]
[24d01h14m42s][HUGECTR][INFO]: Evaluation, AverageLoss: 0.718874
[5917.9, eval_accuracy, 0.718874, 0.5, 1, ]
[24d01h14m42s][HUGECTR][INFO]: Eval Time for 1 iters: 0.002185s
[5917.93, eval_stop, 0.5, ]
[5917.93, train_epoch_end, 1, ]
[5917.95, run_stop, ]
```
